setup.cfg

```diff
@@ -112,6 +112,8 @@ vagrant =
     python-vagrant
 windows =
     pywinrm
+changelog =
+    towncrier
```

Comment:
Also, take into account that it's not a runtime dependency meaning that there's no benefit of exposing this to users.

I didn't state it anywhere in Molecule forums, _I think_, but I usually consider extras as a public API and try to apply other public APIs considerations to extras.